### PR TITLE
Fix port matching for multiple port cases

### PR DESCRIPTION
**Pull request by Sauraus**
The current \\s in the port matching grep is too restrictive, for
example in the case of mysql selinux semanage port rules has the
following entry:

mysqld_port_t                  tcp      1186, 3306, 63132-63164

grep -P '#{new_resource.protocol}\\s+#{new_resource.port}' will never
match causing all operations to fail.

Fixed a minor typo in the install.rb error message and updated
supermarket reference to chef.io

### CODE REVIEW

**Comment by BackSlasher** [comment]
Please split into typo commit and a grep fix commit

About the grep - I'm not very familiar with SELinux and was unaware of one rule containing multiple ports.
Please provide a more specific regex (so that port `1024` won't match `mybestrule1024`). I assume commas and spaces are in order somewhere :)
If we can't, maybe the moment I dread has come and I have to move into custom python code for detection

**Comment by BackSlasher** [comment]
Didn't notice the split. You didn't split as I intended.
All of the obvious fixes should go in one commit, and the behavior-changing fixes (the grep modification in all locations) should go in another.
I might make the obvious fixes myself if I have the time.
Also please rebase against new master.
Ping me when ready :)

**Comment by Sauraus** [comment]
1) What is the value of splitting the commits? Cosmetics?
2) Why rebase when there are no merge conflicts?

I am fine doing the above if there is value to the work, but doing something just for the sake of it being done is no value and thus waste and therefore should not be done.

**Comment by BackSlasher** [comment]
1. Not splitting the commits weakens our ability to troubleshoot later. Let's say I merge the PR. Someone later has an issue with the new regex. Using the log, they see that the fault is in a2a0d59f701410dda034b20c6f011b51e25c6a84. They revert this specific commit, and then their Berkshelf brakes. :( .  
  It also helps me now, since I want to merge the trivial changes you made (Berksfile, my embarrassing typos) regardless of what we decide about the regex. I currently have no manually look at the patch and cut it myself.
2. Rebasing against master (to do FF-merges) is a very common practice where I work. It helps order because the master commit chain ("tree"?) matches the sequence the commits were actually introduced into master.  
  Besides, You master is before 0.6.2 (current version), meaning that if we merge now, your commits will appear before the 0.6.2 bump commit. Very confusing. You can check me on that.  


In short, both things are "more work now for less work later" sort of a thing.
I appreciate your contribution, and if you'd rather focus on the regex expression rather than the git bookskeeping it's fine by me. Worst case scenario I'll do it

**Comment by Sauraus** [comment]
I will be re-submitting this shortly with some other changes as well :)

**Comment by Sauraus** [comment]
I just hope you can keep up with all the pull requests coming your way now. :)

**Comment by BackSlasher** [comment]
Shocked to see like 7 emails from Github. Busy day :)

**Comment by BackSlasher** [comment]
Anyway, we'll handle your contributions this way. **However**, I meant you should split the commits, not the PRs. You can create a branch that has multiple commits and then create a PR from this branch. I handle the merges manually (not via GitHub UI) anyway.
